docs/001_develop/03_client-capabilities/021_desktop-interoperability/001_fdc3-channel-event.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: 'FDC3 Channel Event'
+sidebar_label: 'FDC3 Channel Event'
+id: fdc3-channel-event
+keywords: [interaction, interactive, interoperability, openfin, interop.io, fdc3, fdc3 channel, fdc3 context, data]
+tags:
+- interaction
+- interactive
+- interoperability
+- openfin
+- interop.io
+- fdc3
+- fdc3 channel
+- fdc3 context
+- data
+sidebar_position: 1
+---
+
+# FDC3 Channel Event
+
+A utility element to that listens to an event on its parent element and broadcasts the event detail payload on an FDC3 Channel.
+
+Use cases:
+* Send event on system color channel
+* Send event on app channel
+* Map the event payload before broadcasting on a channel
+
+## Example 1 - Send event on system channel
+
+This will listen to the `rowClicked` event that is fired by the `rapid-grid-pro` element whenever a row is clicked.
+
+It will broadcast on the current system color channel.
+
+It will broadcast with the context type `fdc3.position`.
+
+The context payload will contain the `event.detail.data` payload in the `id` property of the Context object.
+
+```html
+<rapid-grid-pro>
+    <fdc3-channel-event event-name="rowClicked" channel-type="fdc3.position">
+
+    </fdc3-channel-event>
+</rapid-grid-pro>
+```
```

Comment:
Should we specify what the channel broadcast payload will look like in each case?